packages/terra-core-docs/src/terra-dev-site/doc/content-container/example/ContentContainerFill.jsx

```diff
@@ -1,52 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import ContentContainer from 'terra-content-container';
+import Button from 'terra-button';
 import classNames from 'classnames/bind';
+import InputField from 'terra-form-input/lib/InputField';
 import styles from './ContentContainerDocCommon.module.scss';
 
 const cx = classNames.bind(styles);
 
 const display1 = (
   <p key="1">
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus aliquam lectus quis finibus feugiat. Nullam ut sagittis purus.
-    Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut placerat fringilla aliquam.
-    Phasellus in eleifend odio. Nunc maximus sem vel odio lobortis, in accumsan arcu dapibus. Nunc consectetur, nunc in mattis
-    pharetra, augue felis aliquet nunc, non facilisis est velit vel elit.
-  </p>
-);
-const display2 = (
-  <p key="2">
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus aliquam lectus quis finibus feugiat. Nullam ut sagittis purus.
-    Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut placerat fringilla aliquam.
-    Phasellus in eleifend odio. Nunc maximus sem vel odio lobortis, in accumsan arcu dapibus. Nunc consectetur, nunc in mattis
-    pharetra, augue felis aliquet nunc, non facilisis est velit vel elit.
-  </p>
-);
-const display3 = (
-  <p key="3">
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus aliquam lectus quis finibus feugiat. Nullam ut sagittis purus.
-    Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut placerat fringilla aliquam.
-    Phasellus in eleifend odio. Nunc maximus sem vel odio lobortis, in accumsan arcu dapibus. Nunc consectetur, nunc in mattis
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus aliquam
+    lectus quis finibus feugiat. Nullam ut sagittis purus. Pellentesque habitant
+    morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut
+    placerat fringilla aliquam. Phasellus in eleifend odio. Nunc maximus sem vel
+    odio lobortis, in accumsan arcu dapibus. Nunc consectetur, nunc in mattis
```

Comment:
```suggestion
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus aliquam
    lectus quis finibus feugiat. Nullam ut sagittis purus. Pellentesque habitant
    morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut
    placerat fringilla aliquam. Phasellus in eleifend odio. Nunc maximus sem vel
    odio lobortis, in accumsan arcu dapibus. Nunc consectetur, nunc in mattis
```
Since this PR about adding working examples for content container can we get this examples updated with some realistic text as content

Reply:
Updated